win32ss/user/user32/windows/cursoricon.c

```diff
@@ -152,6 +152,10 @@ static BOOL is_dib_monochrome( const BITMAPINFO* info )
     }
 }
 
+/************************************************************************
+ * Return the size of the bitmap info structure including color table and
```

Comment:
```suggestion
/* Return the size of the bitmap info structure including color table and
```
Sure, it's just a nit. But with the additional line we would encourage others to follow that style, or worse: to grow even longer separation-dicks. A whisper in the silence can be louder than a screaming in a steelwork.

Reply:
Changed. Thanks.